docs/src/components/DocsLayout.tsx

```diff
@@ -67,38 +67,75 @@ class DocsLayout extends React.Component<any, any> {
     })
   }
 
-  renderChildren = props => {
-    const { component: Children, render } = this.props
+  renderChildren() {
+    const { children, render } = this.props
+    const sidebarWidth = '270px'
+
+    const treeSectionStyle = {
+      fontWeight: fontWeightBold,
+      margin: '0 0 .5rem',
+      padding: '0 1.2857rem',
+      background: '#201f1f',
+      color: 'white',
+    }
+
+    const treeItemStyle = {
+      padding: '.5em 1.33333333em',
+      textDecoration: 'none',
+      fontSize: '0.85714286em',
+      fontWeight: 400,
+      color: '#ffffff80',
+
+      '& .active': {
+        fontWeight: 'bold',
+      },
+    }
 
     return (
       <>
         <Provider
           theme={mergeThemes(themes.teamsDark, {
             // adjust Teams' theme to Semantic UI's font size scheme
             componentVariables: {
-              MenuDivider: {
-                borderColor: '#ffffff80',
+              TreeItem: {
+                padding: '.5em 1.33333333em',
+                textDecoration: 'none',
+                fontSize: '0.85714286em',
```

Comment:
nit: Can we use `pxToRem()`?

```suggestion
                fontSize: pxToRem(12),
```